tests/testthat/_snaps/dm_to_tibble.md

```diff
@@ -0,0 +1,458 @@
+    Output
+      [[1]]
+      # A tibble: 0 x 3
+      # ... with 3 variables: c <chr>, d <int>, tf_1 <tibble[,1]>
+      
+      [[2]]
+      # A tibble: 1 x 3
+        c            d tf_1$b
+        <chr>    <int> <chr> 
+      1 elephant     2 B     
+      
+      [[3]]
+      # A tibble: 2 x 3
+        c         d tf_1$b
+        <chr> <int> <chr> 
+      1 lion      3 C     
+      2 dog       6 F     
+      
+      [[4]]
+      # A tibble: 2 x 3
+        c         d tf_1$b
+        <chr> <int> <chr> 
+      1 seal      4 D     
+      2 cat       7 G     
+      
+      [[5]]
+      # A tibble: 2 x 3
+        c         d tf_1$b
+        <chr> <int> <chr> 
+      1 seal      4 D     
+      2 cat       7 G     
+      
+    Code
+      print(tbl_keys_from_dm_tf_4$tbl$tf_5)
+    Output
+      [[1]]
+      # A tibble: 0 x 3
+      # ... with 3 variables: k <int>, m <chr>, tf_6 <tibble[,1]>
+      
+      [[2]]
+      # A tibble: 1 x 3
+            k m     tf_6$o
+        <int> <chr> <chr> 
+      1     1 house e     
+      
+      [[3]]
+      # A tibble: 1 x 3
+            k m     tf_6$o
+        <int> <chr> <chr> 
+      1     2 tree  f     
+      
+      [[4]]
+      # A tibble: 1 x 3
+            k m          tf_6$o
+        <int> <chr>      <chr> 
+      1     3 streetlamp h     
+      
+      [[5]]
+      # A tibble: 1 x 3
+            k m          tf_6$o
+        <int> <chr>      <chr> 
+      1     4 streetlamp h     
+      
+
+---
+
+    Code
+      print(serialized_from_tf_4)
+    Output
+      # A tibble: 5 x 6
+        h     i     j        j1 tf_5_json_child             tf_3_json_parent          
+        <chr> <chr> <chr> <int> <chr>                       <chr>                     
+      1 a     three C         3 "{\"empty\":[true],\"data\~ "{\"data\":[{\"g\":\"two\~
+      2 b     four  D         4 "{\"empty\":{},\"data\":[{~ "{\"data\":[{\"g\":\"thre~
+      3 c     five  E         5 "{\"empty\":{},\"data\":[{~ "{\"data\":[{\"g\":\"four~
+      4 d     six   F         6 "{\"empty\":{},\"data\":[{~ "{\"data\":[{\"g\":\"five~
+      5 e     seven F         6 "{\"empty\":{},\"data\":[{~ "{\"data\":[{\"g\":\"five~
+    Code
+      print(serialized_from_tf_4$tf_5_json_child)
+    Output
+      [1] "{\"empty\":[true],\"data\":[{\"k\":1,\"m\":\"a\",\"tf_6_json_parent\":\"{\\\"data\\\":[{}]}\"}]}"                        
+      [2] "{\"empty\":{},\"data\":[{\"k\":1,\"m\":\"house\",\"tf_6_json_parent\":\"{\\\"data\\\":[{\\\"o\\\":\\\"e\\\"}]}\"}]}"     
+      [3] "{\"empty\":{},\"data\":[{\"k\":2,\"m\":\"tree\",\"tf_6_json_parent\":\"{\\\"data\\\":[{\\\"o\\\":\\\"f\\\"}]}\"}]}"      
+      [4] "{\"empty\":{},\"data\":[{\"k\":3,\"m\":\"streetlamp\",\"tf_6_json_parent\":\"{\\\"data\\\":[{\\\"o\\\":\\\"h\\\"}]}\"}]}"
+      [5] "{\"empty\":{},\"data\":[{\"k\":4,\"m\":\"streetlamp\",\"tf_6_json_parent\":\"{\\\"data\\\":[{\\\"o\\\":\\\"h\\\"}]}\"}]}"
+    Code
+      print(serialized_from_tf_4$tf_3_json_parent)
```

Comment:
Can we use `as_glue()` for the column data (or at least for the printing)?

Reply:
done